FishBun/src/main/java/com/sangcomz/fishbun/adapter/image/impl/PicassoAdapter.java

```diff
@@ -29,4 +29,4 @@ public void loadDetailImage(ImageView target, Uri loadUrl) {
                 .centerInside()
                 .into(target);
     }
-}
```

Comment:
Please remove any unnecessary changes.

Reply:
This file have taken it from your project only remember i have deleted picasso realted code

Reply:
Well, do you know why these changes happen?

Reply:
No idea maybe some extra spacing or something had been there before